app/main.py

```diff
@@ -2,0 +1,50 @@
+class CreatingMixin:
+    def creating(self, what_creating: str, what_return: str) -> str:
+        print(f"{self.name} {what_creating}...")
+
+        return what_return
+
+
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    @classmethod
+    def learn_skill(cls, skill: str) -> None:
+        cls.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer, CreatingMixin):
+    skills = ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self) -> str:
+        return self.creating("is creating a webpage",
+                             "<h1>Hello world</h1>")
+
+
+class BackendDeveloper(SoftwareEngineer, CreatingMixin):
+    skills = ["Python", "SQL", "Django"]
+
+    def create_powerful_api(self) -> str:
+        return self.creating("is creating an API",
+                             "http://127.0.0.1:8000")
+
+
+class AndroidDeveloper(SoftwareEngineer, CreatingMixin):
+    skills = ["Java", "Android studio"]
+
+    def create_smooth_mobile_app(self) -> str:
+        return self.creating("is creating a mobile app",
+                             "Ads every three swipes")
+
+
+class FullStackDeveloper(BackendDeveloper, FrontendDeveloper, CreatingMixin):
+    skills = [] + BackendDeveloper.skills + FrontendDeveloper.skills
+
+    def create_web_application(self) -> None:
```

Comment:
The method `create_web_application` is defined to return `None`, but it calls methods that return strings (`create_powerful_api` and `create_awesome_web_page`). If the intention is to use these return values, consider capturing them and returning or processing them within this method.